frontend/src/wizards/Governance/Policy/stable/CM-Configuration-Management/policy-pod.yaml

```diff
@@ -25,15 +25,20 @@ spec:
             - complianceType: musthave
               objectDefinition:
                 apiVersion: v1
-                kind: Pod # nginx pod must exist
+                kind: Pod # httpd pod must exist
                 metadata:
-                  name: sample-nginx-pod
+                  name: sample-httpd-pod
                 spec:
                   containers:
-                    - image: nginx:1.18.0
-                      name: nginx
-                      ports:
-                        - containerPort: 80
+                    - image: registry.redhat.io/rhel9/httpd-24:latest
+                      name: httpd
+                      securityContext:
```

Comment:
is it ok skip 
```
    ports:
    - containerPort: 80
 ```

Reply:
Yes, it just provides information and the container images should specify the ports they expose.